src/biokbase/narrative/tests/job_test_constants.py

```diff
@@ -32,6 +32,13 @@ def generate_error(job_id, err_type):
     return error_strings[err_type]
 
 
+def trim_ee2_state(ee2_state, exclude_fields):
```

Comment:
don't we have this code somewhere else?

Reply:
Yep, `Job._trim_ee2_state`. I just got tired of using that in tests when usually we use independent testing functions